(async ()=> {
    try {
        console.log("Hello World");
        console.log("Justin");
    } catch (err) {
        console.log("Error: ", err)
    }
})();